.travis.yml

```diff
@@ -0,0 +1,5 @@
+language: node_js
```

Comment:
Does travis automatically do a `npm install`?

Reply:
No, I added that into the script